<launch>

	<node pkg="stage_ros" type="stageros" name="stage" args="-g /home/sergi/catkin_ws/src/auction/launch/world_files/5_robots.world"/>
	<include file="$(find auction)/launch/launch_1_robot_vfh.launch">
		<arg name="robot_name" value="robot_0"/>
		<arg name="port" value="#25556"/>
		<arg name="hostname" value="localhost"/>
		<arg name="load_capacity" value="3"/>
		<arg name="max_linear_speed" value="4"/>
	</include>
	<include file="$(find auction)/launch/launch_1_robot_vfh.launch">
		<arg name="robot_name" value="robot_1"/>
		<arg name="port" value="#25557"/>
		<arg name="hostname" value="localhost"/>
		<arg name="load_capacity" value="1"/>
		<arg name="max_linear_speed" value="3"/>
	</include>
	<include file="$(find auction)/launch/launch_1_robot_vfh.launch">
		<arg name="robot_name" value="robot_2"/>
		<arg name="port" value="#25558"/>
		<arg name="hostname" value="localhost"/>
		<arg name="load_capacity" value="1"/>
		<arg name="max_linear_speed" value="5"/>
	</include>
	<include file="$(find auction)/launch/launch_1_robot_vfh.launch">
		<arg name="robot_name" value="robot_3"/>
		<arg name="port" value="#25559"/>
		<arg name="hostname" value="localhost"/>
		<arg name="load_capacity" value="3"/>
		<arg name="max_linear_speed" value="4"/>
	</include>
	<include file="$(find auction)/launch/launch_1_robot_vfh.launch">
		<arg name="robot_name" value="robot_4"/>
		<arg name="port" value="#25560"/>
		<arg name="hostname" value="localhost"/>
		<arg name="load_capacity" value="7"/>
		<arg name="max_linear_speed" value="1"/>
	</include>

</launch>